packages/ag-charts-community/src/chart/series/hierarchy/hierarchySeries.ts

```diff
@@ -79,10 +79,10 @@ export abstract class HierarchySeries<S extends SeriesNodeDatum> extends Series<
     colorKey?: string = undefined;
 
     @Validate(OPT_COLOR_STRING_ARRAY)
-    fills?: string[] = undefined;
+    fills: string[] = ['#c16068', '#a2bf8a', '#ebcc87', '#80a0c3', '#b58dae', '#85c0d1'];
```

Comment:
Is this pushing the default theme fills as a fallback in? If so, could we use a constant defined by the default theme to capture this relationship?

Reply:
> If an invalid colour is provided, please default to the theme, or hard-coded default theme colours instead of not rendering anything.

I didn't find a way to achieve this - but this matches the behaviour of the pie series

Reply:
Discussed offline.